client/src/pages/Search/AddressSearch.js

```diff
@@ -2,8 +2,23 @@ import { IconButton, InputAdornment, TextField } from "@mui/material";
 import { usePlacesWidget } from "react-google-autocomplete";
 import { HUNTER_COLLEGE_ADDRESS } from "./constants";
 import ClearIcon from "@mui/icons-material/Clear";
+import getUserCoordinates from "../../utils/getUserCoordinates";
+import axios from "axios";
+import MyLocationIcon from "@mui/icons-material/MyLocation";
+import { useState } from "react";
 
 export default function AddressSearch({ updateFields }) {
```

Comment:
pass the updateQuerySearch function down here so we can reflect the address changes